### DISCUSSION TITLE
env variables without prefixes

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#16306] [Questions]
How to use custom `env` variables without prefixes `NUXT` or `NUXT_PUBLIC`
i just want write something like this:
```ts
import type { NuxtConfig } from 'nuxt/config';

export default {
	runtimeConfig: {
		public: {
			TEST: process.env.TEST,
		},
	},
} as NuxtConfig;
```

and in production mode use command `TEST=test node ./build/server/index.mjs`, but i have `undefined`

**Comment by armezit** [comment]
@mihail727 I tried many ways to achieve what you're looking for, but no luck!
Seems only prefixed vars work.

Build your project and then take a look at the `_runtimeConfig` and `overrideConfig()` in node server file. (Usually at the `.output/server/chunks/nitro/node-server.mjs` path)

The `_runtimeConfig` contains config items from build time. The `overrideConfig()` function replace those items by any existing prefixed env variable.
